practice-tool/src/widgets/character_stats.rs

```diff
@@ -22,34 +37,52 @@ impl Stats for CharacterStatsEdit {
                 Datum::int("Intelligence", &mut s.intelligence, 1, 99),
                 Datum::int("Faith", &mut s.faith, 1, 99),
                 Datum::int("Arcane", &mut s.arcane, 1, 99),
-                Datum::int("Souls", &mut s.runes, 1, i32::MAX),
-            ]
-            .into_iter()
-        })
+                Datum::int("Souls", &mut s.runes, 0, i32::MAX),
+                Datum::byte("Scadutree Blessing", &mut b.scadutree, 0, if has_blessings { 20 } else { -1 }),
+                Datum::byte("Revered Spirit Ash", &mut b.revered_spirit_ash, 0, if has_blessings { 10 } else { -1 }),
```

Comment:
I see now how this relates to my comment in https://github.com/veeenu/practice-tool-core/pull/19. We should _never_ use sentinel values to represent states as that has a lot of potential for confusion. Addressing my comment above should make this check unnecessary as the iterator is pre-filtered.

Reply:
The thing is that I wanna hide blessings values from stats editor when the version is older than DLC release (or make them disabled at least), coz memory addresses would be invalid here.
If not using unsafe block, I have to do something more complex to implement global temporary variables(e.g. add a member field to CharacterStats)